torchvision/models/inception.py

```diff
@@ -70,10 +70,10 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False):
 
     def forward(self, x):
         if self.transform_input:
-            x = x.clone()
-            x[:, 0] = x[:, 0] * (0.229 / 0.5) + (0.485 - 0.5) / 0.5
-            x[:, 1] = x[:, 1] * (0.224 / 0.5) + (0.456 - 0.5) / 0.5
-            x[:, 2] = x[:, 2] * (0.225 / 0.5) + (0.406 - 0.5) / 0.5
+            x_ch0 = torch.unsqueeze(x[:, 0], 1) * (0.229 / 0.5) + (0.485 - 0.5) / 0.5
+            x_ch1 = torch.unsqueeze(x[:, 1], 1) * (0.224 / 0.5) + (0.456 - 0.5) / 0.5
+            x_ch2 = torch.unsqueeze(x[:, 2], 1) * (0.225 / 0.5) + (0.406 - 0.5) / 0.5
+            x = torch.cat((x_ch0, x_ch1, x_ch2), 1)
```

Comment:
Thanks for the PR!

Please correct me if I'm wrong, but I believe there might be a simpler (and more efficient) way of implementing this functionality.

What about the following:

```python
device = x.device
dtype = x.dtype
scale = torch.tensor([
    0.229 / 0.5,
    0.224 / 0.5,
    0.225 / 0.5], dtype=dtype, device=device)
bias = torch.tensor([
    (0.485 - 0.5) / 0.5,
    (0.456 - 0.5) / 0.5,
    (0.406 - 0.5) / 0.5], dtype=dtype, device=device)
scale = scale.view(1, -1, 1, 1)
bias = bias.view(1, -1, 1, 1)

x = x * scale + bias
```

Reply:
Indeed, that seems like a much better approach.
Thanks!